src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -91,7 +91,13 @@ const FormDefinitionDetailPage = () => {
       <Stack gap={2}>
         <Grid container spacing={2}>
           <Grid item xs={12} md={8}>
-            <CommonCard title='Details'>
+            <CommonCard
+              title={
+                <Typography variant='h4' component='h2' sx={{ mb: 2 }}>
+                  Details
+                </Typography>
+              }
+            >
```

Comment:
This doesn't seem ideal.. the point of CommonCard is to hide these style details. Is there a reason we have `h5` CommonCards some places but this page in particular needs `h4` cards? We should discuss this w design

Reply:
Hm, you're right -- I was just trying to figure out a way to update the Form Definition page to match the designs, without messing with CommonCards across the app -- but we should definitely take a broader view.

I started a thread in the figjam here: https://www.figma.com/board/axGI9vwmuBRC4uZ11ZJ3RP?node-id=773-10818#858654148

My guess is that Bobby made the change from <h5> to <h4> in the card titles on this page so that the cards could have subtitles at the <h5> level, such as Form Applicability - Rules and Projects. So as not to block this PR on that conversation, I'll revert the change here to use <h5> and then make those sub-headings <h6>, if that sounds good to you.

Reply:
Here's a preview of how that approach looks:

![Screenshot 2024-07-01 at 12 30 29 PM](https://github.com/greenriver/hmis-frontend/assets/16002775/18fe3770-6911-436d-8839-d9e4dcb88c39)

Reply:
okay sounds good, thanks for following up there